migrations/v9-MIGRATION.md

```diff
@@ -0,0 +1,23 @@
+# V89 API Migration Guide
+
+This latest release updates the SDK to use BillingClient 8. This version removed APIs to query for expired subscriptions and consumed one-time products, aside from other improvements. You can check the full list of changes here: https://developer.android.com/google/play/billing/release-notes#8-0-0
```

Comment:
> This latest release updates

I would change to

> This release updates

or to 

> The V9 release updates

Now it's the latest, but eventually it won't be

Reply:
And also, to make it extremely clear that it's not our SDK changing this, I would change
> This version removed APIs

to 

> This version of BillingClient removed APIs